docs/release-strategy.md

```diff
@@ -47,19 +55,38 @@ The Release Manager for each release is identified in the Description of the Mil
 
 The following are the steps for how Y-stream and Z-stream releases gets cut.
 
-### Y-Stream
+### Y-Stream alpha pre-release (`X.Y.0a1`)
+
+1. Determine a commit on the main branch that will serve as the basis for the next release - most of the time this should be the latest commit.
+1. Validate the release branch with an [E2E test](ci.md).
+1. Create a new release on GitHub targeting the release branch and using the latest Y-Stream tag as the previous release (e.g. `0.15.1` precedes `0.16.0a1`). If there have been other pre-release, then use the previous pre-release.
+1. Announce release via the following:
+    - The `#announce` channel on Slack
+    - The `announce` mailing list
```

Comment:
Do we want to do as broad of an announcement for pre-releases as we do regular releases?